docs/AboutUs.adoc

```diff
@@ -4,53 +4,52 @@
 :imagesDir: images
 :stylesDir: stylesheets
 
-AddressBook - Level 4 was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team. +
-_{The dummy content given below serves as a placeholder to be used by future forks of the project.}_ +
-{empty} +
-We are a team based in the http://www.comp.nus.edu.sg[School of Computing, National University of Singapore].
+Event Manager is based on Address-Book Level 4, which was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team. +
+We are a team of Y2 Computer Engineering undergraduate students based in the http://www.comp.nus.edu.sg[School of Computing, National University of Singapore].
 
 == Project Team
 
-=== John Doe
-image::damithc.jpg[width="150", align="left"]
-{empty}[http://www.comp.nus.edu.sg/~damithch[homepage]] [https://github.com/damithc[github]] [<<johndoe#, portfolio>>]
+=== Chia Xiang Rong
+image::chiaxr.png[width="150", align="left"]
+{empty}[http://github.com/chiaxr[github]] [<<johndoe#, portfolio>>]
 
-Role: Project Advisor
+Role: Developer +
+Responsibilities: RSVP/Attendance
 
 '''
 
-=== John Roe
-image::lejolly.jpg[width="150", align="left"]
-{empty}[http://github.com/lejolly[github]] [<<johndoe#, portfolio>>]
+=== Gerald Chua Deng Xiang
+image::geraldcdx.png[width="250", align="left"]
+{empty}[https://www.comp.nus.edu.sg/~geraldc/website2-0/index.html[homepage]] [https://github.com/Geraldcdx[github]] [<<GCDX#, portfolio>>]
 
-Role: Team Lead +
-Responsibilities: UI
+Role: Developer +
+Responsibilities: Comment Section Architect
 
 '''
 
-=== Johnny Doe
-image::yijinl.jpg[width="150", align="left"]
-{empty}[http://github.com/yijinl[github]] [<<johndoe#, portfolio>>]
+=== James Arista Yaputra
+image::jamesyaputra.png[width="150", align="left"]
+{empty}[http://github.com/jamesyaputra[github]] [<<jamesyaputra#, portfolio>>]
 
 Role: Developer +
-Responsibilities: Data
+Responsibilities: User Authentication
 
 '''
 
-=== Johnny Roe
-image::m133225.jpg[width="150", align="left"]
-{empty}[http://github.com/m133225[github]] [<<johndoe#, portfolio>>]
+=== Chuang Qin Kai
+image::cqinkai.png[width="150", align="left"]
+{empty}[http://github.com/cqinkai[github]] [<<johndoe#, portfolio>>]
 
 Role: Developer +
-Responsibilities: Dev Ops + Threading
+Responsibilities: Event Reminders
 
 '''
 
-=== Benson Meier
-image::yl_coder.jpg[width="150", align="left"]
-{empty}[http://github.com/yl-coder[github]] [<<johndoe#, portfolio>>]
+=== Ho Phi Long
+image::P_Long.jpg[width="150", align="left"]
```

Comment:
@Tertium3 do change your image name to the mentioned format: <username_in_lower_case>.png   . At the current stage, your image won't appear in the project list [page](https://nuscs2113-ay1819s1.github.io/website/admin/projectList.html)